### PR TITLE
8257516: define test group for manual tests

### PR DESCRIPTION
**Pull request by frkator**
Defined new test groups as defined in ticket. @fguallini

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8257516](https://bugs.openjdk.java.net/browse/JDK-8257516): define test group for manual tests

### Reviewers
 * [Igor Ignatyev](https://openjdk.java.net/census#iignatyev) (@iignatev - **Reviewer**)

### Download
`$ git fetch https://git.openjdk.java.net/jdk pull/1416/head:pull/1416`
`$ git checkout pull/1416`

### CODE REVIEW

File: test/jdk/TEST.groups [hunk 1]
```diff
@@ -323,7 +323,8 @@ jdk_core = \
     :jdk_security \
     :jdk_text \
     :core_tools \
-    :jdk_other
+    :jdk_other \
+    :jdk_core_manual
```
Comment (AlanBateman) [inline test/jdk/TEST.groups:327]: Please don't add manual tests to jdk_core.

Comment (frkator) [reply test/jdk/TEST.groups]: Removed.

File: test/jdk/TEST.groups [hunk 2]
```diff
@@ -323,8 +323,7 @@ jdk_core = \
     :jdk_security \
     :jdk_text \
     :core_tools \
-    :jdk_other \
-    :jdk_core_manual
+    :jdk_other
```
Comment (iignatev) [inline test/jdk/TEST.groups:326]: it would seem you have introduced a trailing space

File: test/jdk/TEST.groups (hunk 1 above)
Comment (iignatev) [reply test/jdk/TEST.groups]: aren't they already a part of `jdk_core` test group? e.g. `java/net/HugeDataTransferTest.java` is in `:jdk_net` (as `jdk_net` includes `java/net` directory), and `:jdk_core` includes `:jdk_net`?

**Review by iignatev — CHANGES_REQUESTED** [review]
@frkator, you will need to open a new JBS ticket for this change.

**Comment by frkator** [comment]
@iignatev as for:
"https://github.com/openjdk/jdk/pull/1416#discussion_r532904345"
accidentaly resolved. Yes they are but the goal is to list all manual tests under one label for counting purposes.
Could you please assist with above?
I assume it was due to some intermittent issue, I was able to build on the failed ones.

**Comment by iignatev** [comment]
> @iignatev as for:
> "[#1416 (comment)](https://github.com/openjdk/jdk/pull/1416#discussion_r532904345)"
> accidentaly resolved. Yes they are but the goal is to list all manual tests under one label for counting purposes.

I understand that; my comment was rather to @AlanBateman 's request not to add manual tests into `:jdk_code`. they already are, and removing explicit inclusion of `:jdk_core_manual` to `:jdk_core` won't change that.

-- Igor

**Review by iignatev — APPROVED** [review]
LGTM

**Pull request closed without merging**